### PR TITLE
Symbols mapping, state persist in URL

### PR DESCRIPTION
**Pull request by SaenkoM**
Simplifies currency handling (symbols and pairs)
Fixes tests
Fixes derivatives and tax reports snapshot fetching
Adds filters persist in URL
Enables sync on each app load
Adds ability to choose displayed columns for each section with filters

### CODE REVIEW

File: src/components/FundingOfferHistory/FundingOfferHistory.columns.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default function getColumns(props) {
       copyText: rowIndex => filteredData[rowIndex].symbol,
     },
     {
-      id: 'amount',
+      id: 'amountOrig',
```
Comment (ezewer) [inline src/components/FundingOfferHistory/FundingOfferHistory.columns.js]: Why was this changed?

File: src/components/Orders/Orders.columns.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ export default function getColumns(props) {
       copyText: rowIndex => filteredData[rowIndex].type,
     },
     {
-      id: 'amount',
+      id: 'amountOrig',
```
Comment (ezewer) [inline src/components/Orders/Orders.columns.js]: Same question here

File: src/components/Positions/Positions.columns.js [hunk 3]
```diff
@@ -194,7 +194,7 @@ export default function getColumns(props) {
     },
     ...ACTIVE_POSITIONS_COLS,
     {
-      id: 'swap',
+      id: 'marginFunding',
```
Comment (ezewer) [inline src/components/Positions/Positions.columns.js]: why this changed?

File: src/components/Positions/Positions.columns.js [hunk 4]
```diff
@@ -212,7 +212,7 @@ export default function getColumns(props) {
       copyText: rowIndex => filteredData[rowIndex].marginFunding,
     },
     {
-      id: 'swapType',
+      id: 'marginFundingType',
```
Comment (ezewer) [inline src/components/Positions/Positions.columns.js]: Why this changed?

Comment (SaenkoM) [reply src/components/Positions/Positions.columns.js]: reused already existing list of columns https://github.com/bitfinexcom/bfx-report-ui/pull/322/files#diff-7b76267d52037735bda922404034396aR56, but in some cases the id's didn't correspond to data attributes

**Comment by ezewer** [comment]
> probably a good idea to reduce the PR size down a bit

Totally agree, @SaenkoM lets try to keep each PR as small as possible and iterate several times, is simpler as to review and find bugs.
